rel/ensmallen-release.sh

```diff
@@ -39,6 +39,9 @@ if [ "$#" -eq "4" ]; then
   sed -i 's/ENS_VERSION_NAME[ ]*\".*\"$/ENS_VERSION_NAME \"'"$4"'\"/' include/ensmallen_bits/ens_version.hpp;
 fi
 
+# update CONTRIBUTING.md
+sed -i "s/ensmallen-[0-9]*\.[0-9]*\.[0-9]*/ensmallen-$MAJOR.$MINOR.$PATCH/g" CONTRIBUTING.md;
+
 git pull
 git add include/ensmallen_bits/ens_version.hpp;
```

Comment:
Don't forget to add `CONTRIBUTING.md` here also. :+1:

Reply:
Ok, adding it :)